discovery/src/main/scala/akka/discovery/SimpleServiceDiscovery.scala

```diff
@@ -54,6 +55,23 @@ object SimpleServiceDiscovery {
         case None => host
       }
   }
+
+}
+
+/**
+ * A service lookup. It is up to each mechanism to decide
+ * what to do with the optional portName and protocol fields.
```

Comment:
I don't think `portName` and `protocol` are self describing so would be good to clarify what those are and examples of such, using things like Akka Cluster (remoting), gRPC, HTTP.

Reply:
:+1: